deeppavlov/vocabs/wiki_sqlite.py

```diff
@@ -28,21 +28,25 @@ class WikiSQLiteVocab(SQLiteDataIterator):
     Args:
         data_url: an URL where to download a DB from
         data_dir:  a directory where to save downloaded DB to
+        join_docs: whether to join extracted docs with ' ' or not
 
-    """
+    Attributes:
+        join_docs: whether to join extracted docs with ' ' or not
 
-    def __init__(self, data_url: str, data_dir: str = '', **kwargs):
+    """
 
-        super().__init__(data_dir=data_dir, data_url=data_url)
+    def __init__(self, data_url: str, data_dir: str = '', join_docs: bool=True, **kwargs):
+        super().__init__(data_dir=data_dir, data_url=data_url, shuffle=kwargs.get('shuffle', False))
```

Comment:
Why not add `shuffle=False` to `__init__()` parameters then?

Reply:
ok